src/services/security/modules/RedefineModule/index.ts

```diff
@@ -150,7 +164,7 @@ export class RedefineModule implements SecurityModule<RedefineModuleRequest, Red
           severity: redefineSeverityMap[issue.severity.label],
         })),
         balanceChange: result.data?.balanceChange,
-        simulation: result.data?.simulation,
+        simulation: isObjectEIP712TypedData(data) ? undefined : result.data?.simulation,
```

Comment:
Is there no `result.data.simulation` for EIP712 scans?

Reply:
There is, but we only use the simulation ID to render a link to Redefine's dashboard. when those simulation IDs are opened on the Redefine dashboard for EIP-712 scans they only show "No transaction found". 
As long as that is not fixed on their end we should hide the link to their dashboard.